### DISCUSSION TITLE
Videodownloadhelper Icon Not Present

### DISCUSSION THREAD

**Discussion by astorg6** [aclap-dev/video-downloadhelper#514] [Help and Q&A]
### Your question

I'm using VDH in Firefox. The VDH icon is now not present and I am not able to download any videos. The addon webiste says my VDH is loaded and active. It says I am using VDH version 8.2.2.8. I just uninstalled Firefox, and then installed it again. The deleted VDH and reinstalled it too. What can I do?

### Addon & CoApp version and details

VDH version: 8.2.2.8

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user, like you.

First off, you failed to follow the instructions on the page where you created your problem report.  You need to always supply that information by following those instructions.  You can correct you post.  Click the 3 dots above & to the right of your post.

You should also be using the latest versions of the add-on & the CoApp.  Click the Wiki link at the top of this page.  The Wiki is a sort of user manual.  You should get in the habit of frequently referring to that.  In the Wiki, look for the link to the beta versions of VDH & the link to CoApp installation.  Make sure you are using the latest versions.

As for the issue of the missing icon, that has been raised many times before.  Do a search of this forum.  You'll find your answer.  Then remember to search this forum before you post.  Always.  More than likely, somebody else has encountered your problem.  That is true in this case & is likely to be true in future.

**Comment by astorg6** [comment]
Thanks very much for your answer.I did most of the things you suggested already and did search the forum.But I guess I'm not very good at it.
I'll go again, and I very much appreciate your reply.Thanks................Astor

    On Monday, May 20, 2024 at 03:33:49 AM GMT+1, GrampaWildWilly ***@***.***> wrote:  
 
 


I do not work for VDH. I'm just another user, like you.

First off, you failed to follow the instructions on the page where you created your problem report. You need to always supply that information by following those instructions. You can correct you post. Click the 3 dots above & to the right of your post.

You should also be using the latest versions of the add-on & the CoApp. Click the Wiki link at the top of this page. The Wiki is a sort of user manual. You should get in the habit of frequently referring to that. In the Wiki, look for the link to the beta versions of VDH & the link to CoApp installation. Make sure you are using the latest versions.

As for the issue of the missing icon, that has been raised many times before. Do a search of this forum. You'll find your answer. Then remember to search this forum before you post. Always. More than likely, somebody else has encountered your problem. That is true in this case & is likely to be true in future.

**Comment by paulrouget** [comment]
Do this: https://github.com/aclap-dev/video-downloadhelper/wiki/Lost-icon

**Comment by astorg6** [comment]
Hi Paul.I did as your link suggested but still no icon.I'm heading back to the forum to do another search and see.But I'm not very good at this, so we'll see.
Thanks very much for your reply. I very much appreciate it.....................Astor

    On Monday, May 20, 2024 at 10:41:28 AM GMT+1, Paul ***@***.***> wrote:  
 
 


Do this: https://github.com/aclap-dev/video-downloadhelper/wiki/Lost-icon

**Reply by paulrouget** [reply]
When you click on the Extension popup:

![image](https://github.com/aclap-dev/video-downloadhelper/assets/368367/9bfb43ce-2d25-4e6f-943d-235652f85037)

If you don't see VDH icon, it means VDH is not installed.
